test/sql/alter.sql

```diff
@@ -0,0 +1,42 @@
+\ir include/create_single_db.sql
```

Comment:
Should we also have some test for dropping and recreating a column with the same name for both tests below? I imagine we want both one column that's just dropped and one that's dropped and recreated. Also do we want to make modifications do ddl_ops_1 and ddl_ops 2 to stress this?